pom.xml

```diff
@@ -17,11 +17,10 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-        <camel.version>4.8.1</camel.version>
+        <camel.version>4.9.0</camel.version>
```

Comment:
thanks. I will take a look soon

this will directly address the OIDC issues, or is there more to it?

Reply:
yes, the OIDC support is included in 4.9.0. Last piece missing is to make this configurable on the pre-built images. I am going to add some source config via EnvVars similar to what we have with TLS support so you can enable/disable OIDC on integration source via just configuration. Will be a follow-up PR